packages/arb-token-bridge-ui/tests/e2e/specs/urlQueryParam.cy.ts

```diff
@@ -0,0 +1,244 @@
+/* eslint-disable cypress/no-unnecessary-waiting */
```

Comment:
Can we remove those now that we use `waitUntil`?

Reply:
nice catch, done!